static/app/views/performance/transactionSummary/transactionOverview/charts.tsx

```diff
@@ -161,6 +167,23 @@ function TransactionSummaryCharts({
     organization,
     location
   );
+  // For mep-incompatible displays hide event count
+  const hideTransactionCount =
+    canUseMetricsInTransactionSummary(organization) && display === DisplayModes.TREND;
+
+  // For partially-mep-compatible displays show event count as a %
+  const showTransactionCountAsPercentage =
+    canUseMetricsInTransactionSummary(organization) && display === DisplayModes.LATENCY;
+
+  function getTotalValue() {
+    if (totalValue === null || hideTransactionCount) {
+      return <Placeholder height="24px" />;
+    }
+    if (showTransactionCountAsPercentage && unfilteredTotalValue) {
+      return formatPercentage(totalValue / unfilteredTotalValue);
+    }
+    return totalValue.toLocaleString();
+  }
```

Comment:
Nit: it's probably better to separate this out instead of having an inline function, but that can be a follow up.